Matlab/Matlab.sublime-syntax

```diff
@@ -145,11 +145,11 @@ contexts:
       scope: punctuation.separator.continuation.matlab
   brackets:
     - match: '\['
-      scope: meta.brackets.matlab
+      scope: punctuation.section.brackets.begin.matlab
       push:
         - meta_content_scope: meta.brackets.matlab
```

Comment:
I just noticed this, sorry for having more and more tweaks!

For `meta.` scopes, we apply them to the tokens that enter and exit the scope. So we want the `[` and `]` to get `meta.brackets.matlab`.

To do this, just use `meta_scope` instead of `meta_content_scope`.